packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -110,7 +110,10 @@ _Props/Attrs_
 -   `isEmpty` - is a read-only property that indicates whether the editor is empty or not. This will be achieved through Tiptap's
     [isEmpty](https://tiptap.dev/api/editor#is-empty) API. The component and the Angular directive will have a getter method
     that can be used to bind it in the Angular application.
--   `fitToContent` - is a boolean attribute allows the text area to expand vertically to fit the content.
+-   `fit-to-content` - is a boolean attribute allows the text area to expand vertically to fit the content.
+-   `placeholder` - is an attribute to include a placeholder text for the editor when it is empty. This can be achieved through
```

Comment:
Can you state explicitly is the placeholder treated as markdown or a plain string?

Reply:
Updated